<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAction</class>
 <widget class="QWidget" name="AddAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Voice Action</string>
  </property>
  <widget class="QPushButton" name="button_add">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Action</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_command">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>170</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Command:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_path">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>210</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Path to file:</string>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
